cognite/seismic/protos/types.proto

```diff
@@ -283,3 +283,11 @@ enum Dimensions {
     TWO_DEE = 2;
     THREE_DEE = 3;
 }
+
+enum TraceHeaderField {
+    ENERGY_SOURCE_POINT = 0;
```

Comment:
I'd still like to have an UNSPECIFIED here. Just because we've been bitten by defaults in protobufs in the past...

Reply:
(Alright, as long as it's only used in a repeated field we can detect that it isn't set, but if we use it in non-repeated fields we might get bitten.)

Reply:
Added UNSPECIFIED just in case..

Reply:
Or some other name to avoid the conflicts.. :(